CHANGELOG.md

```diff
@@ -1,5 +1,36 @@
 # [Versions](https://mui.com/versions/)
 
+## v5.16.1
+
+_Jul 10, 2024_
+
+A big thanks to the 3 contributors who made this release possible. Here are some highlights ✨:
+
+- ⚛️ Material UI is now compatible with React 18.3.1
```

Comment:
Should we mention other packages? Truth is we updated to React 18.3.1 everywhere.

Reply:
I think we should, what about:

```suggestion
- ⚛️ All packages, including Material UI, are now compatible with React 18.3.1
```

Reply:
Applied